make a proper tcl module
Index: debian-gpib/linux-gpib-user/language/tcl/ibCmds.c
===================================================================
--- debian-gpib.orig/linux-gpib-user/language/tcl/ibCmds.c
+++ debian-gpib/linux-gpib-user/language/tcl/ibCmds.c
@@ -8,7 +8,7 @@
 
 void ib_CreateVerboseError(Tcl_Interp *interp,char *entry );
 
-int Gpib_tcl_Init ( Tcl_Interp *interp ){
+int Gpib_Init ( Tcl_Interp *interp ){
 
 
 extern int gpibCmd _ANSI_ARGS_(( ClientData clientData,
@@ -21,6 +21,7 @@ extern int gpibCmd _ANSI_ARGS_(( ClientD
 		         (ClientData) NULL,
 			 (Tcl_CmdDeleteProc *) NULL );
 
+    Tcl_PkgProvide (interp, "gpib", "4.3.5");
     return TCL_OK;
 }
 
Index: debian-gpib/linux-gpib-user/language/tcl/examples/example1.tcl
===================================================================
--- debian-gpib.orig/linux-gpib-user/language/tcl/examples/example1.tcl
+++ debian-gpib/linux-gpib-user/language/tcl/examples/example1.tcl
@@ -5,7 +5,7 @@
 #
 #
 
-load ../libgpib_tcl.so
+package require gpib
 
 #
 # init device
Index: debian-gpib/linux-gpib-user/language/tcl/examples/example2.tcl
===================================================================
--- debian-gpib.orig/linux-gpib-user/language/tcl/examples/example2.tcl
+++ debian-gpib/linux-gpib-user/language/tcl/examples/example2.tcl
@@ -12,8 +12,8 @@
 #  clausi@chemie.fu-berlin.de
 #
 
-load ../libgpib_tcl.so
-load ../../../contrib/tcltk/modules/libBLT.so.2.1
+package require gpib
+package require BLT
 
 set auto_path [linsert $auto_path 0 ./tclib]
 
